packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -75,8 +74,6 @@ export const runSessionExpiryKey = "Fluid.GarbageCollection.RunSessionExpiry";
 export const disableSessionExpiryKey = "Fluid.GarbageCollection.DisableSessionExpiry";
 // Feature gate key to write the gc blob as a handle if the data is the same.
 export const trackGCStateKey = "Fluid.GarbageCollection.TrackGCState";
```

Comment:
Should this be a new value or is that a separate change?

Reply:
Since the feature itself did not change, it would be a separate change if the feature turned out to be broken.

Reply:
Oh, I thought this was the broken one. Never mind.